README.md

```diff
@@ -374,7 +374,7 @@ SELECT partman.run_maintenance('public.history');
 pg_partman has a function `run_maintenance` that allows one to automate the table maintenance.
 
 ```
-SELECT run_maintenance(p_analyze := false);
+SELECT partman.run_maintenance(p_analyze := false);
```

Comment:
Good catch! And yes if you are copying and pasting this could definitely throw you off. It should be clear cut for anyone. Thanks!